osu.Game/Overlays/MusicController.cs

```diff
@@ -354,6 +350,10 @@ private void changeTrack()
                 if (queuedTrack == CurrentTrack)
                 {
                     AddInternal(queuedTrack);
+
+                    double normalizedVol = current.BeatmapInfo.AudioNormalization?.IntegratedLoudnessInVolumeOffset ?? 0.8;
+                    this.TransformBindableTo(audioNormalizationManager.TrackNormalizeVolume, normalizedVol, 300, Easing.Out);
+
```

Comment:
Add back the logger (see my other comment as to why)
```suggestion
                    Logger.Log("Normalization Status: " + (current.BeatmapInfo.AudioNormalization?.IntegratedLoudness != null
                        ? $"on ({Math.Round((double)(current.BeatmapInfo.AudioNormalization?.IntegratedLoudnessInVolumeOffset * 100))}%)"
                        : "off"));
```